fileserve.rb

```diff
@@ -0,0 +1,51 @@
+# my lame attempt at imitating the File Explorer API
```

Comment:
What was being attempted here? Is this file still needed?

Reply:
Oh, is this part of what you were talking about, when you were thinking of expanding the file editor app to have more functionality than just "edit the file at this path"?

Reply:
We can discuss further, but the scope of the file editor app is just a utility app (think microservice) for editing files, and meant to be invoked only from other apps.

Reply:
Well, then we'll have to figure out a mechanism for the invoking app to give a URL for downloading and uploading the file being edited. Right now it is hard-coded as something like /pun/sys/files/api/v1/<path> and would surely break if this were installed at another center unless they had the same configuration as us.